src/server.ts

```diff
@@ -5,7 +5,10 @@ const port = 3002
 const app = express()
 
 app.get('*', (req, res) => {
-  const { html } = token.generateTokenHtml({ secret: '906090' })
+  const { html } = token.generateTokenHtml({
+    secret: '906090',
+    domain: 'apple.com',
```

Comment:
Maybe we should use something else, I think apple won't be happy about this one :)
How about `bwl.gg`?

Reply:
It's custom data that pass when you run `yarn start`. But anyway, I changed it